<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>862</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#add_btn{
	background-image: url(:/icons/icons/update.png);
	background: ;
	border: none;
}

#add_btn:hover{
	border: 1px solid rgb(252, 233, 79);
}

#out_btn{
	
	background-image: url(:/icons/icons/close2.png);
	background: ;
	border: none;
}

#out_btn:hover{
	border: 1px solid rgb(252, 233, 79);
}

#close_btn{
	
	
	background-image: url(:/icons/icons/closes2.png);
	background: ;
	border: none;
}

#close_btn:hover{
	border: 1px solid rgb(252, 233, 79);
}

#delete_btn{
	
	background-image: url(:/icons/icons/delete2.png);
	background: ;
	border: none;
}

#delete_btn:hover{
	border: 1px solid rgb(252, 233, 79);
}

#save_btn{
	
	background-image: url(:/icons/icons/save2.png);
	background: ;
	border: none;
}

#save_btn:hover{
	border: 1px solid rgb(252, 233, 79);
}

#help_btn{
	
	background-image: url(:/icons/icons/help2.png);
	background: ;
	border: none;
}

#help_btn:hover{
	border: 1px solid rgb(252, 233, 79);
}

#line_status{
background:;
border:none;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="add_btn">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>450</y>
      <width>89</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>450</y>
      <width>631</width>
      <height>62</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_name">
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_phone">
           <property name="text">
            <string>Phone</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLineEdit" name="line_name"/>
         </item>
         <item>
          <widget class="QLineEdit" name="line_phone"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_last">
           <property name="text">
            <string>Last Name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_date">
           <property name="text">
            <string>Date</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLineEdit" name="line_last"/>
         </item>
         <item>
          <widget class="QLineEdit" name="line_date"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="tableItems">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>731</width>
      <height>431</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Last Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Phone</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Date</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="out_btn">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>40</y>
      <width>89</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="close_btn">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>120</y>
      <width>89</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_btn">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>200</y>
      <width>89</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="save_btn">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>280</y>
      <width>89</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="help_btn">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>360</y>
      <width>89</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_status">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>260</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>862</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionDelete"/>
    <addaction name="actionClose"/>
    <addaction name="actionOpen"/>
    <addaction name="actionEdit"/>
    <addaction name="actionClear_BD"/>
   </widget>
   <widget class="QMenu" name="menuSession">
    <property name="title">
     <string>Session</string>
    </property>
    <addaction name="actionClose_session"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionFAQ"/>
    <addaction name="actionAbout"/>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSession"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionEdit"/>
   <addaction name="actionSave"/>
   <addaction name="actionSave_as"/>
   <addaction name="actionDelete"/>
   <addaction name="actionFAQ"/>
   <addaction name="actionAbout"/>
   <addaction name="actionHelp"/>
   <addaction name="actionClose_session"/>
   <addaction name="actionClose"/>
  </widget>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normalon>:/icons/icons/save.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset>
     <normalon>:/icons/icons/saveas.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset>
     <normalon>:/icons/icons/delete.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionClose_session">
   <property name="icon">
    <iconset>
     <normalon>:/icons/icons/close.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Close session</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset>
     <normalon>:/icons/icons/closes.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionFAQ">
   <property name="icon">
    <iconset>
     <normalon>:/icons/icons/faq.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>FAQ</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset>
     <normalon>:/icons/icons/about.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset>
     <normalon>:/icons/icons/help.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <normalon>:/icons/icons/open.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="icon">
    <iconset>
     <normalon>:/icons/icons/edit.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
  <action name="actionClear_BD">
   <property name="text">
    <string>Clear BD</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
